src/de/duenndns/ssl/MemorizingTrustManager.java

```diff
@@ -619,8 +619,12 @@ private static void setLatestEventInfoReflective(Notification notification,
 	@TargetApi(Build.VERSION_CODES.HONEYCOMB)
 	void startActivityNotification(Intent intent, int decisionId, String certName) {
 		Notification notification;
-		final PendingIntent call = PendingIntent.getActivity(master, 0, intent,
-				0);
+		final PendingIntent call;
+		if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.S) {
```

Comment:
I would factor out the API relevant part instead of the whole `getActivity()` call.

```java
int flags = 0;
if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.S) {
     flags |= PendingIntent.FLAG_MUTABLE
}
call = PendingIntent.getActivity(master, 0, intent, flags);
```

Reply:
Actually, are you sure that the flag must be mutable and not immutable? I don't see why the pending intent's data should be mutable.

If it's immutable then we could simply unconditionally append `FLAG_IMMUTABLE`: